tools/python/boutiques/bosh.py

```diff
@@ -62,16 +62,18 @@ def execute(*params):
                             help="Runs the container as local user ({0}) instead of root.".format(os.getenv("USER")))
         results = parser.parse_args(params)
         descriptor = results.descriptor
+        inp = results.invocation
 
         # Do some basic input scrubbing
-        inp = results.invocation
         if not os.path.isfile(inp):
             raise SystemExit("Input file {} does not exist".format(inp))
         if not inp.endswith(".json"):
             raise SystemExit("Input file {} must end in json".format(inp))
         if not os.path.isfile(descriptor):
             raise SystemExit("JSON descriptor {} does not exist".format(descriptor))
 
+        valid = invocation(descriptor, '-i', inp)
+        valid = invocation(descriptor, '-i', inp)
```

Comment:
why is this line repeated twice?

Reply:
shall we catch the exception and exit gracefully? Since it's a command-line tool and not the API?

Reply:
Typo, thanks :) 

I don't really understand what you mean in the latter point? For example, the last statement: scripts will normally exit if a file is not found.

Reply:
if the descriptor is not valid, `invocation` will throw an exception. We may want to catch it and print a neat error message on stderr instead of throwing the exception at the user.

Reply:
Ah, I understand. I've made invocation print in the same form as the validate module